pyerrors/fits.py

```diff
@@ -763,7 +768,7 @@ def chisqfunc(p):
 
     if output.method != 'Levenberg-Marquardt':
         if output.method == 'migrad':
-            tolerance = 1e-4
+            tolerance = 1e-1 # default value set by iminuit
```

Comment:
Hi Pia, I just had a quick look at your changes, thanks! Is there a reason why you changed the migrad solver tolerance? In the past I have seen problems with using `1e-1`, that's why I lowered it back then.

Reply:
No, I just reset it.